src/index.html

```diff
@@ -8,9 +8,39 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <div>
+        <label data-qa="keypress">
+          <input
+            class="form-big form"
+            type="text"
+            name="search_big"
+            data-qa="big"
```

Comment:
```suggestion
            data-qa="keypress"
```

it can help with test error